src/picongpu/include/simulation_defines/param/communication.param

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2013-2015 Axel Huebl, Felix Schmitt, Heiko Burau, Rene Widera
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#pragma once
+
+
+namespace picongpu
+{
+
+/* define unique communication id's
+ *
+ * define unique id's for all elements those can be send and received
+ */
+enum CommunicationTag
+{
+    NO_COMMUNICATION = 0u,
+    FIELD_B = 1u,
+    FIELD_E = 2u,
+    FIELD_J = 3u,
+    FIELD_JRECV = 4u,
+    FIELD_TMP = 5u,
+    PAR_ELECTRONS = 16u,
```

Comment:
I would prefer just defining a `PAR_FIRSTTAG` here and adding for each species an index.

with that, nothing in this file should be changed by a user any more and due to that, I am not sure a param is necessary for it.